tests/validators/test_uuid.py

```diff
@@ -0,0 +1,164 @@
+        ('1b4e28ba-2fa1-11d2-883f-0016d3cca427', UUID('1b4e28ba-2fa1-11d2-883f-0016d3cca427')),
+        ('6ba7b810-9dad-11d1-80b4-00c04fd430c8', UUID('6ba7b810-9dad-11d1-80b4-00c04fd430c8')),
+        ('886313e1-3b8a-5372-9b90-0c9aee199e5d', UUID('886313e1-3b8a-5372-9b90-0c9aee199e5d')),
+        ('c0a8f9a8-aa5e-482b-a067-9cb3a51f5c11', UUID('c0a8f9a8-aa5e-482b-a067-9cb3a51f5c11')),
+        (b'\x12\x34\x56\x78' * 4, UUID('12345678-1234-5678-1234-567812345678')),
+        (b'\x00\x00\x00\x00' * 4, UUID('00000000-0000-0000-0000-000000000000')),
+        # Invalid UUIDs
+        (
+            'not-a-valid-uuid',
+            Err(
+                'Input should be a valid UUID, invalid character: expected an optional prefix of'
+                + ' `urn:uuid:` followed by [0-9a-fA-F-], found `n` at 1'
+            ),
+        ),
+        (
+            '12345678-1234-5678-1234-5678123456789',
+            Err('Input should be a valid UUID, invalid group length in group 4: expected 12, found 13'),
+        ),
+        (
+            '12345678-1234-1234-1234-1234567890123',
+            Err('Input should be a valid UUID, invalid group length in group 4: expected 12, found 13'),
+        ),
+        (b'\x00\x00\x00\x000' * 4, Err('Input should be a valid UUID, invalid length: expected 16 bytes, found 20')),
+        ('550e8400-e29b-41d4-a716', Err('Input should be a valid UUID, invalid group count: expected 5, found 4')),
+        (
+            'f47ac10b-58cc-4372-a567-0e02b2c3d47',
+            Err('Input should be a valid UUID, invalid group length in group 4: expected 12, found 11'),
+        ),
+        (
+            'de305d54-75b4-431b-adb2-eb6b9e54601',
+            Err('Input should be a valid UUID, invalid group length in group 4: expected 12, found 11'),
+        ),
+        (
+            '1b4e28ba-2fa1-11d2-883f-0016d3cca42',
+            Err('Input should be a valid UUID, invalid group length in group 4: expected 12, found 11'),
+        ),
+        (
+            '6ba7b810-9dad-11d1-80b4-00c04fd430c',
+            Err('Input should be a valid UUID, invalid group length in group 4: expected 12, found 11'),
+        ),
+        (
+            '886313e1-3b8a-5372-9b90-0c9aee199e5',
+            Err('Input should be a valid UUID, invalid group length in group 4: expected 12, found 11'),
+        ),
+        (
+            'c0a8f9a8-aa5e-482b-a067-9cb3a51f5c1',
+            Err('Input should be a valid UUID, invalid group length in group 4: expected 12, found 11'),
+        ),
+    ],
+)
+def test_uuid(input_value, expected):
+    v = SchemaValidator({'type': 'uuid'})
+    if isinstance(expected, Err):
+        with pytest.raises(ValidationError, match=re.escape(expected.message)):
+            result = v.validate_python(input_value)
+            print(f'input_value={input_value} result={result}')
+    else:
+        output = v.validate_python(input_value)
+        assert output == expected
```

Comment:
Potentially add `assert isinstance(output, UUID)` here and all the other successful validations?

Reply:
done